packages/flutter_tools/templates/app/android.tmpl/app/src/main/AndroidManifest.xml.tmpl

```diff
@@ -11,7 +11,7 @@
         android:icon="@mipmap/ic_launcher">
         <activity
             android:name=".MainActivity"
-            android:launchMode="singleTop"
+            android:launchMode="singleTask"
```

Comment:
do you mind also changing the plugin example app template?

Reply:
This should be resolved in https://github.com/flutter/flutter/pull/56899/commits/7a4dcd31ac5cd3464eaa02186f6fd3f26a910b94

Reply:
actually, there's no such a thing as a "plugin example app template". Thanks for updating the module though